examples/view_3d_map.html

```diff
@@ -71,9 +71,8 @@
                     new itowns.ElevationLayer(config.id, config),
                 ).then(debugMenu.addLayerGUI.bind(debugMenu));
             }
-            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
             itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
-
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
```

Comment:
This inversion is due to #2000. It should be done also on other examples with multiple DEM until #2000 is solved.

Reply:
I changed it in all the other examples where the 2 elevation layers were added.